src/BloomBrowserUI/templates/customXMatter/MXBPamphlet-XMatter/MXBPamphlet-XMatter.less

```diff
@@ -25,25 +25,12 @@
 
 .insideBackCover {
     .marginBox {
-        .bloom-editable:only-child {
```

Comment:
It seems clear from that that the intent was to make the bloom-editable children of the margin box divide the vertical space using flexbox's auto mode (or just use all of it if there is only one). Since we didn't see any change from applying the same rules to translationGroup, it seems that our default is doing something similar. But maybe we should check whether there cases where setting the margin box and translationGroups to flex would help? Maybe it would better handle cases where one of the boxes has more than a half-page of content and the other has less?